README.md

```diff
@@ -62,6 +62,9 @@ Fields:
  - `enabledFeatures`. Type: \[String\]. Optional. Default value: empty array. Used on the front end to determine which opt-in features the institution has access to. (Either make this required, or make `disabledFeatues` optional. If this field is left optional, remove default value.)
  - `facebookId`: Type: String. Optional. The id of the facebook page using our facebook app that is associated with the institution.
  - `filterOn` Type: Boolean. Optional. The field that distinguished open bots (value `false`) from closed bots (value `true`). (To do: move this to `dialogSettings`. Make required.)
+ - `freeze` Type: Object. Optional. Enable and control "freezing" the bot when an admin sends a message.  See AH-1054
```

Comment:
Looking at this again, I suspect this would be better as "freezeOnAdminResponse" or something more descriptive.  Thoughts?

Reply:
I guess there's a trade-off; calling it something super-specific makes it more confusing if we ever repurpose it (I wish we hadn't called `twilioErroCode` that), but it also makes it more informative. I don't see us doing anything very different with this particular field in the near future, so yeah, I guess the more descriptive name is good.

Reply:
True; it might be worth spending a minute thinking about how else we might want to use this.

Reply:
Changed to `freezeOnAdminResponse` for now.